test/e2eTests/tests/dashboard_test.js

```diff
@@ -3,47 +3,177 @@ Feature('Dashboard');
 
 const LOG_TAG = '[DashboardTests]';
 
+let Patient = null;
+
 // In order to be on the dashboard we must first login
 Before(async (I) => {
   I.login();
 });
 
+After(async (I, Apis, Data) => {
+  // Removes the visit if it was created
+  const cleanUpPatientVisit = async (patient) => {
```

Comment:
I thought you were creating a util function?

Reply:
I'm working on it.